include/hal/library/responder/setcertlib.h

```diff
@@ -48,8 +53,11 @@ extern bool libspdm_write_certificate_to_nvm(
 #endif
     uint8_t slot_id, const void * cert_chain,
     size_t cert_chain_size,
-    uint32_t base_hash_algo, uint32_t base_asym_algo);
-
+    uint32_t base_hash_algo, uint32_t base_asym_algo
+#if LIBSPDM_SET_CERT_PARAMS
+    , bool *need_reset, bool *is_busy
```

Comment:
I am wondering if we need `is_busy` for CSR API as well.

Reply:
Yes probably. I can do that in a separate pull request, but let me rename `LIBSPDM_SET_CERT_PARAMS` to something more generic so that CSR can leverage it as well.

Reply:
Can we file a separate issue? I feel #2893 only handles INVALID_REQUEST.